INSTALL.rst

```diff
@@ -238,3 +238,20 @@ Conda packages
 
 The conda packaging scripts for Matplotlib are available at
 https://github.com/conda-forge/matplotlib-feedstock.
+
+Docker image
+======================
+
+The official docker image for Matplotlib is avaliable at
+
+- mrakitin/mpl\-docker \- Docker Hub (https://hub.docker.com/r/mrakitin/mpl-docker)
```

Comment:
I think it's too early to declare it as an official MPL docker image from my docker hub. @tacaswell had an idea to place it to the GitHub packages, and this work will have to be done on my side. I hope it will not take too long to make it available, and then the corresponding docker commands can be updated.